content/posts/newsletter/2020-05-01.md

```diff
@@ -0,0 +1,191 @@
+
+Goffi released [Salut à Toi progress note for calendar week 21](https://www.goffi.org/b/kmebCba3m38dr9sgzgGrkp/progress-note)!
+
+[Poezio](https://poez.io) released 0.13.0 and 0.13.1 which now requires python 3.7. These versions bring Poezio into the club of OMEMO supporting clients and also enable HTTP upload via dedicated plugins. Check the full [changelog](https://lab.louiz.org/poezio/poezio/-/blob/master/CHANGELOG) to look at all the improvements of these versions.
+
+[Biboumi](https://biboumi.louiz.org/), the XMPP/IRC gateway, was released in version 0.8.5, fixing compilation issues with GCC 10.
+
+The python based XMPP automatic file sender [Errol has been updated to v2.0.1](https://blog.agayon.be/errol_update1.html) which modernizes its async I/O calls and adds more privacy. By moving from 'aionotify' to the better maintained 'watchdog', Errol theoretically extends its support (from Linux only) to Windows, Mac OS X and FreeBSD.
+
+## Services
+
+blabber.im now [supports IPv6](https://blabber.im/blabber-im-ist-nun-ipv6-tauglich/). The service operators created an [invitation link which works for their server but also for every other ID](https://blabber.im/en/i-moved/). If you moved to an XMPP based chat client, you can now conveniently put this link into the status of your previous non-XMPP chat clients. This link offers your contacts all the necessary infos to be able to reach you and (if they want to) follow you into the XMPP world.
+
+![Jabber.org](https://www.jabber.org/jabber.png)
+
+jabber.org migrates its server to Prosody. [Stay informed!](https://www.jabber.org/notices.html)
+
+yax.im rolled out [new anti-spam measures](https://yaxim.org/blog/2020/05/12/new-anti-spam-measures/). You may be interested in the '[The Jabber Spam Fighting Manifesto](https://github.com/JabberSPAM/jabber-spam-fighting-manifesto)', too.
+
+## Libraries
+
+The XMPP library for Python 3.7+, [Slixmpp](https://lab.louiz.org/poezio/slixmpp), was [updated to version 1.5.0](https://mail.jabber.org/pipermail/jdev/2020-May/090424.html), with minor releases 1.5.1 and 1.5.2. The [changelog]( https://lab.louiz.org/poezio/slixmpp/-/tags/slix-1.5.0) shows improvements of numerous XEPs and preliminary support for OMEMO.
+
+Quite some time has past since the last official Smack release. Now there are some very early alpha releases of Smack 4.4.
+Among additional support for more than a dozen new XEPs, the new release will contain a new architecture for modular transports, as well as a modified API that applies the builder pattern to stanza types.
+A date for the final release is not yet known, but it is now time to give feedback to make sure to get your changes into the next stable Smack release!
+[Read the detailed list of changes].(https://github.com/igniterealtime/Smack/wiki/Smack-4.4-Readme-and-Upgrade-Guide).
```

Comment:
`[Read the detailed list of changes](https://github.com/igniterealtime/Smack/wiki/Smack-4.4-Readme-and-Upgrade-Guide).
`
=> Remove the dot: "`].(`"